pkg/controller/common/helm.go

```diff
@@ -25,7 +25,8 @@ func init() {
 }
 
 const (
-	istioVersion = "v1.1"
+	DefaultMaistraVersion = "v1.1"
+	LegacyMaistraVersion  = "v1.0"
```

Comment:
Curious to know if we can keep different supported versions in ENV. which might be useful in other places as well.

Reply:
These are just constants to avoid using '"v1.0"' or '"v1.1"' throughout the code.  The "other places" are used to configure the correct 'NetworkAttachmentDefinition' on the mesh members, depending on which version of the mesh they are using.

'DefaultMaistraVersion' aligns with the latest version supported by the operator.  This is the version that will be used if the user does not specify a version in their SMCP resource.

'LegacyMaistraVersion' aligns with SMCP resources that do not have a version specified because they were managed by an older version of the operator.  This allows the new operator to initialize the new field with the correct value.  (SMCP resources managed by the new operator will have the field properly initialized.)